pkg/controller/internalreleaseimage/templates/worker/files/root-ca.yaml

```diff
@@ -0,0 +1,5 @@
+mode: 0644
+path: "/etc/pki/ca-trust/source/anchors/root-ca.crt"
```

Comment:
Same comment as above applies, preferrably the naming on-disk is indicative of what it's used for, since root-ca is a bit of an overloaded term